opensearch/src/test/java/org/opensearch/sql/opensearch/config/TestConfig.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+package org.opensearch.sql.opensearch.config;
+
+import com.google.common.collect.ImmutableMap;
+import java.util.Map;
+import org.opensearch.sql.DataSourceSchemaName;
+import org.opensearch.sql.analysis.symbol.Namespace;
+import org.opensearch.sql.analysis.symbol.Symbol;
+import org.opensearch.sql.analysis.symbol.SymbolTable;
+import org.opensearch.sql.data.type.ExprCoreType;
+import org.opensearch.sql.data.type.ExprType;
+import org.opensearch.sql.exception.ExpressionEvaluationException;
+import org.opensearch.sql.expression.Expression;
+import org.opensearch.sql.expression.ReferenceExpression;
+import org.opensearch.sql.expression.env.Environment;
+import org.opensearch.sql.planner.logical.LogicalPlan;
+import org.opensearch.sql.planner.physical.PhysicalPlan;
+import org.opensearch.sql.storage.StorageEngine;
+import org.opensearch.sql.storage.Table;
+
+/**
+ * Configuration will be used for UT.
+ */
+public class TestConfig {
+  public static final String INT_TYPE_NULL_VALUE_FIELD = "int_null_value";
+  public static final String INT_TYPE_MISSING_VALUE_FIELD = "int_missing_value";
+  public static final String DOUBLE_TYPE_NULL_VALUE_FIELD = "double_null_value";
```

Comment:
Do you need all the stuff here? Probably, null/missing references could be removed.